### DISCUSSION TITLE
Real Time Logs drop drastically after 9am EST but look normal the next day after upgrade

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#11578] [2.4]
### Version

2.4.20

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

upgrading

### Installation Type

Distributed

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64gig

### Storage for /

500gig

### Storage for /nsm

8tb

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

 est
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/ec442347-d942-4f09-9866-f6b575d1b405)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/e51745d9-ddc1-4e73-8eda-58e1260e7dae)

Every day when I login at 7 am everything looks normal then promptly at 9 am since the 9.4.20 upgrade logging tanks! 
but when I come back the next day and look at the 24 hour trend it looks normal

I don't understand why during the day it looks like nothing is happening. Does anyone have any thoughts on this?

13:43est 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/467140d6-0090-4da5-a36d-2f594395fec0)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/aa02c561-ce15-4fd8-b638-8ec3e56d3398)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by jaypfe** [comment]
Does the SOC behave the same way?

**Reply by Phillip-Samson** [reply]
I don't know 
I don't usually look in there as much

This is the status this morning 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/aed89e6d-5f78-417f-9c5c-0545abd96f92)
same things is happening

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/06a97186-8190-458d-a859-50cbde80e8d1)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/de7c9fb7-f638-405e-8fc7-d394ee9b060f)

**Comment by weslambert** [comment]
> ### Version
> 2.4.20
> 
> ### Installation Method
> Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.
> 
> ### Description
> upgrading
> 
> ### Installation Type
> Distributed
> 
> ### Location
> cloud
> 
> ### Hardware Specs
> Exceeds minimum requirements
> 
> ### CPU
> 16
> 
> ### RAM
> 64gig
> 
> ### Storage for /
> 500gig
> 
> ### Storage for /nsm
> 8tb
> 
> ### Network Traffic Collection
> other (please provide detail below)
> 
> ### Network Traffic Speeds
> Less than 1Gbps
> 
> ### Status
> Yes, all services on all nodes are running OK
> 
> ### Salt Status
> No, there are no failures
> 
> ### Logs
> No, there are no additional clues
> 
> ### Detail
> est ![image](https://user-images.githubusercontent.com/36277967/276702887-ec442347-d942-4f09-9866-f6b575d1b405.png)
> 
> ![image](https://user-images.githubusercontent.com/36277967/276704246-e51745d9-ddc1-4e73-8eda-58e1260e7dae.png)
> 
> Every day when I login at 7 am everything looks normal then promptly at 9 am since the 9.4.20 upgrade logging tanks! but when I come back the next day and look at the 24 hour trend it looks normal
> 
> I don't understand why during the day it looks like nothing is happening. Does anyone have any thoughts on this?
> 
> 13:43est ![image](https://user-images.githubusercontent.com/36277967/276703959-467140d6-0090-4da5-a36d-2f594395fec0.png) ![image](https://user-images.githubusercontent.com/36277967/276705051-aa02c561-ce15-4fd8-b638-8ec3e56d3398.png)
> 
> ### Guidelines
> * [x]  I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

Have you checked the Redis queue?

It might be possible that traffic volume is higher during the day, and logs could be getting backed up and not processed until later in the day.

**Reply by Phillip-Samson** [reply]
first thing I checked
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/50272f28-bc6f-4f2c-8e37-ae55abb38591)
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/a1e1089d-fa26-4dfe-87db-d20eb99a4cc1)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/992389b1-1208-42b6-bd42-53716ba40cc8)


Is there something in the config that would cause this type of behavior

**Reply by Phillip-Samson** [reply]
Same today. Is it possible the forward node is queuing these up or something?


![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/b5092582-a384-4ab7-90c1-abf8dfb28fad)

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/4f3ea06b-b75b-4974-87f1-13595bf4ae30)

**Reply by Phillip-Samson** [reply]
@weslambert 
Any thoughts or next steps

**Reply by Phillip-Samson** [reply]
Complete mystery. I have millions of logs that will be here tomorrow but my Redis queue is at 0 . What am I missing
Its as if the forward node is not sending the logs in 
. How do I test for that??
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/36277967/0e431460-f654-4600-ae12-54e9d3cc3ec6)

**Reply by weslambert** [reply]
Are the date and time correct on each machine?
